### PR TITLE
Clarify difference between message mode and HTTP content mode

### PR DESCRIPTION
**Pull request by cneijenhuis**
Another attempt to fix https://github.com/cloudevents/spec/issues/645

See also https://github.com/cloudevents/spec/pull/660

Signed-off-by: Christoph Neijenhuis <christoph.neijenhuis@commercetools.de>

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -136,6 +136,9 @@ a stand-alone event format and stored in the message body.
 A "binary-mode message" is one where the event data is stored in the message
 body, and event attributes are stored as part of message meta-data.
 
+While protocol bindings typically use "binary-mode messages" directly on the
```
Comment (grant) [inline spec.md:139]: Not sure if this note belongs in the spec. It seems more like a usage/user guide detail.

**Comment by cneijenhuis** [comment]
FYI I won't make todays meeting due to travel, and will be on vacation for the two weeks after.

I'm ok if someone else takes over, or putting it on hold while I'm gone. Sorry for the inconvenience!

**Comment by duglin** [comment]
Approved on the 7/30 call w/o these lines: https://github.com/cloudevents/spec/pull/672/files#diff-958e7270f96f5407d7d980f500805b1bR139-R140

@cneijenhuis there was some concern about those lines... both from a perspective that it wasn't clear what it was trying to address, and some folks didn't think it was necessary.

Would you be ok with this PR w/o those lines? If so, we can merge. But if you'd like to keep them then we should it more in the PR and on next week's call.

**Comment by duglin** [comment]
@cneijenhuis ping! :-)

**Comment by cneijenhuis** [comment]
I'm back from vacation and have removed those lines. Thanks for your patience! 😉

**Comment by duglin** [comment]
@cneijenhuis thanks and welcome back!

Merging.

**Pull request merged**